modules/gapi/samples/bind_before_serialization_example.cpp

```diff
@@ -0,0 +1,17 @@
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/s11n.hpp>
+#include <opencv2/gapi/garg.hpp>
+
+int main()
+{
+    std::cout << "This sample is non-complete. It is used as code snippents in documentation." << std::endl;
+// ! [bind usage]
+    std::vector<cv::GRunArgP> graph_outs;
+    cv::GRunArgs out_args;
+
+    for (auto &&out : graph_outs) {
+        out_args.emplace_back(cv::gapi::bind(out));
+    }
+    const auto sargsout = cv::gapi::serialize(out_args);
+// ! [bind usage]
+}
```

Comment:
You could keep all examples in the single file, no need to introduce many.

Also, this file could be `api_ref_snippets`. Just put more snippets there.

Reply:
Done